CI/syntaxCheck.py

```diff
@@ -26,7 +26,7 @@
 else:
     # Run Check on App Examples
     passAppEx = 1
-    for package in appExamples.keys():
+    for package in list(appExamples.keys()):
```

Comment:
I would expect that this will need to be rewritten as the app examples are moved to be integrated in the library but for now, it's ok